.github/workflows/builder_bazel_slsa3.yml

```
@@ -0,0 +1,69 @@
+# Copyright 2023 SLSA Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+permissions: {}
+
+on:
+  workflow_call:
+    inputs:
+      rekor-log-public:
+        description: "Allow publication of your repository name on the public Rekor log"
+        required: false
+        type: boolean
+        default: false
+      targets:
+        description: "Artifacts to build"
+        required: true
+        type: string
+      flags:
+        description: "Flags for the bazel build command"
+        required: false
+        type: string
+        default: ""
+
+    outputs:
+      attestations-download-name:
+        description: >
+          Name of the artifact to download all the attestations.
+
+          When run on a `pull_request` trigger, attestations are not signed and have an ".intoto" extension.
+          When run on other triggers, attestations are signed and have an "intoto.sigstore" extension.
+        value: ${{ jobs.slsa-run.outputs.attestations-download-name }}
+jobs:
```

nit: blank line between sections.
```suggestion
        value: ${{ jobs.slsa-run.outputs.attestations-download-name }}

jobs:
```